rho_playbook.yml

```diff
@@ -22,6 +22,7 @@
     - virt
     - virt_what
     - jboss
+    - jboss-scan
```

Comment:
I've never liked the role name of `jboss` when its more focused on *EAP*. If we are naming/renaming things could we update it to have an eap focus?

Reply:
Good point! I'll rename the fact to `jboss.eap.*` as we discussed. The folder can be something like `jboss-eap` or `jboss_eap`, unless you have a preference here.